app/code/Magento/Customer/Api/CustomerRepositoryInterface.php

```diff
@@ -15,7 +15,7 @@
 interface CustomerRepositoryInterface
 {
     /**
-     * Create or update a customer.
+     * use POST to Create customer and PUT to Update customer
```

Comment:
I prefer to revert this change, as this interface used not only for web api, but also as API for external modules.